registry.json

```diff
@@ -151,6 +151,20 @@
     "operatorName": "CONRAD",
     "github": "conraddit"
   },
+  {
+    "ticker": "BEGIN",
+    "poolId": "b59bf139701b8cdae5c997225f438db7603ef1d9fd9a1f62dc5821eb",
+    "social": {
+      "website": "https://begin.is"
```

Comment:
```suggestion
      "website": "https://begin.is",
```

Reply:
I cannot update this, but you missed a comma in the json and validation failed. Please fix